components/cms-modern/AdaptiveImage/AdaptiveImage.tsx

```diff
@@ -32,7 +34,7 @@ const AdaptiveImage: React.SFC<AdaptiveImageProps> = (props) => {
         return null;
     }
 
-    const defaultImageUrl = getImageURL(image, transformations);
+    const defaultImageUrl = getImageURL(image, transformations)
```

Comment:
A bunch of the other lines in this file have a semicolon, what is it meant to be for this project?